lib/ansible/modules/crypto/openssl_privatekey.py

```diff
@@ -111,6 +116,13 @@
         default: auto
         choices: [ auto, cryptography, pyopenssl ]
         version_added: "2.8"
+    backup:
+        description:
+            - Create a backup file including the timestamp information so you can get
+              the original private key back if you somehow clobbered it incorrectly.
```

Comment:
```suggestion
              the original private key back if you created a new one by accident.
```

Reply:
Hmm, maybe this is even better:
```suggestion
              the original private key back if you overwrote it with a new one by accident.
```
What do you think?

Reply:
Yeah, as a non-native "clobbering" was a bit too complex for my taste. :-)